python/examples/advanced/Image_Logging.ipynb

```diff
@@ -0,0 +1,282 @@
+          ]
+        }
+      ],
+      "source": [
+        "import os\n",
+        "from PIL import Image\n",
+        "\n",
+        "from whylogs.extras.image_metric import log_image\n",
+        "\n",
+        "img1 = Image.effect_mandelbrot((256, 256), (-3, -2.5, 2, 2.5), 9)\n",
+        "img2 = Image.effect_mandelbrot((256, 256), (-3, -2.5, 2, 2.5), 20)\n",
+        "\n",
+        "results = log_image(img1)\n",
+        "print(results.view().get_column(\"image\").to_summary_dict())"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "kd9IqAfXIzX4"
+      },
+      "source": [
+        "As you can see above, just passing in an `Image` results in a column named \"image\" in the profile. You can pass in a list of images, which will append an index to each column name:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 31,
+      "metadata": {
+        "id": "uioACRBVKzxX",
+        "outputId": "0738803d-9b8a-45e4-8daf-258afd328528",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 205
+        }
+      },
+      "outputs": [
+        {
+          "output_type": "error",
+          "ename": "AttributeError",
+          "evalue": "ignored",
+          "traceback": [
+            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+            "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
+            "\u001b[0;32m<ipython-input-31-3b41b984c632>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0mresults\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mlog_image\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mimg1\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mimg2\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mresults\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mview\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget_column\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"image_1\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mto_summary_dict\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
+            "\u001b[0;31mAttributeError\u001b[0m: 'NoneType' object has no attribute 'to_summary_dict'"
```

Comment:
Do we know what these errors are about?

Reply:
Yes, there's a but in `log_image()` that's fixed in this PR. I haven't figured out how to run the notebook with the fix though...

Reply:
Not sure how you are running it, but I just pulled down and built your PR and this notebook runs without error, so I think its good to go, but you should delete the cell output as it makes it look like it doesn't work.

You might consider showing the images, e.g.

`display(img1)
display(img2)`